CONTRIBUTING.md

```diff
@@ -11,11 +11,22 @@ $ go version
 go version devel go1.18-[...]
 ```
 **Note:** If using gotip and `go version` does not return version 1.18, you may need to symbolic link `gotip` to `go`.
-```
-ls -s path_to_gotip /user/local/bin/go
+```sh
+$ ln -s `which gotip` /user/local/bin/go
 ```
 Use `which gotip` to get the path to gotip.
 
+## Clonning
+
+After forking FerretDB you can clone for best practices use the following instruction:
```

Comment:
```suggestion
After forking FerretDB you can clone the repository - for rgw best practices use the following instructions:
```

Just slight grammar fix, hope you don't mind :)

Reply:
Sure, thanks! :)